analyzer/protocol/stun/stun.evt

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
```

Comment:
Can you also fix uses of tabs for alignment here as well?

Reply:
There isn't a tab here.

Reply:
I meant in this file.

Reply:
I did.  They are all tabs.